pymyq/request.py

```diff
@@ -179,11 +179,11 @@ async def _send_request(
                 )
                 if err.status == 401:
                     raise err
-                if err.status == 429:
-                    timeout = int(err.headers.get("Retry-After", TOO_MANY_REQUEST_TIMEOUT)) if err.headers is not None else TOO_MANY_REQUEST_TIMEOUT
-                    _LOGGER.debug("Too many request have been made - putting a temporary pause on sending any requests for %d minutes", timeout/60)
-                    self._block_request_until = datetime.utcnow() + timedelta(seconds=timeout)
-                    raise UserRateLimit(f"Got 429 error - stopping request  until {str(self._block_request_until)}. there were {self._request_made} request") from err
+                # if err.status == 429:
+                #     timeout = int(err.headers.get("Retry-After", TOO_MANY_REQUEST_TIMEOUT)) if err.headers is not None else TOO_MANY_REQUEST_TIMEOUT
+                #     _LOGGER.debug("Too many request have been made - putting a temporary pause on sending any requests for %d minutes", timeout/60)
+                #     self._block_request_until = datetime.utcnow() + timedelta(seconds=timeout)
+                #     raise UserRateLimit(f"Got 429 error - stopping request  until {str(self._block_request_until)}. there were {self._request_made} request") from err
```

Comment:
If always avoid the 429 works - check your logs, see if something else ends up happening - because then we would like to automatically do whatever that is when we get a 429. With ignoring the 429s - how long does your system run?

Reply:
I'm running with this branch. It's the first one that has stayed connected for more than 18 minutes. I'm currently at 68 minutes.

I had about 20 minutes of 429's before I got connected, but no 429's in the last 68 minutes.

Reply:
It's been running for about 3 hours without issue. There are no 429's in the logs for 2.5 hours now.

Reply:
Can you try with this commit instead @jarrodmoss https://github.com/Python-MyQ/Python-MyQ/pull/9/commits/66fbe4749ed023112ccf8f5b98998365b8dbb479

Takes some of your changes

Reply:
It works for me, and with a couple of tweaks in the pull request I just sent, it stays online without dropping for a few seconds.